src/stravalib/protocol.py

```diff
@@ -264,77 +471,6 @@ def resolve_url(self, url: str) -> str:
             )
         return url
 
-    def _request(
```

Comment:
i didn't delete or modify much in this method. It looks that way because of the new methods and where I added them.